### DISCUSSION TITLE
Visual studio 2022: device not found

### DISCUSSION THREAD

**Discussion by wannestal** [nanoframework/Home#1351] [Q&A]
I took all required steps (imo) to start programming an ESP32, but the device is not found:

14:51:20.679 [NanoDevices: new device arrival COM3]
14:51:20.693 [NanoDevices: candidate nano device COM3]
14:51:20.790 [NanoDevices: checking device  COM3 @ 921600]
14:51:21.826 [NanoDevices: checking device  COM3 @ 460800]
14:51:22.833 [NanoDevices: checking device  COM3 @ 115200]
14:51:25.811 [NanoDevices: checking device  COM3 *** 2nd attempt ***]
14:51:25.817 [NanoDevices: checking device  COM3 @ 921600]
14:51:27.832 [NanoDevices: checking device  COM3 @ 460800]
14:51:29.851 [NanoDevices: checking device  COM3 @ 115200]
14:51:32.523 [NanoDevices: quitting device COM3]
14:51:32.524 [NanoDevices: Serial device enumeration completed. Found 0 devices]

In device manager I see no issues at Silicon Labs CP210c USB to UART Bridge (COM3). Bitrate for the port is set to 115200.

I have also tried to quit and reinstall the extension for VS 2022. All most recent versions and drivers are installed.

What could be checked further to come to a solution?

KR
Wannes

**Comment by josesimoes** [comment]
What command did you use to flash your ESP32? And what what the output?

(suggestion for a more interactive support, please join our [Discord community](https://discordapp.com/invite/gCyBu8T). )

**Reply by wannestal** [reply]
Hi, I dont know why, but today everything is ok. I can run/debug a program on my ESP32. This is good, but unfortunately I have no root cause of the issue I had.

KR
Wannes
